.gitignore

```diff
@@ -5,7 +5,7 @@
 .byebug_history
 .DS_Store
 .elasticbeanstalk/*
-.env.*
+.env
```

Comment:
can you make this rule ignore everything with `.env.*` other than `.env.template`? I use files with secrets that this was ignoring!

Reply:
True! will do.

Reply:
Also, now you'll need to copy over the contents to `.env` whenever you need to use them. I changed it because I noticed that it .env was constantly overriding my `.env.development` values.

Reply:
Ah right, good point - thanks :)